recipes/app_config.rb

```diff
@@ -94,3 +94,8 @@
   mode '0644'
   variables directory: "#{shared_log_directory}"
 end
+
+execute "reschedule daily crontab" do
+  command "sed -i 's/25 6   * * */25 20   * * */' /etc/crontab"
+  only_if "grep '25 6   * * *' /etc/crontab 2>&1 >/dev/null"
```

Comment:
can we make grep by cron name? ie `daily.cron`

Reply:
Noted, change grep by cron name